src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -125,14 +133,25 @@ public long bnf() {
         throw new UnsupportedOperationException("bnf() not yet implemented");
     }
 
-    // @todo #15:30min Implement details() by parsing the string representation
-    //  of transaction according to the pattern, described in the white
-    //  paper. Replace relevant test case with actual tests.
     @Override
-    public String details() {
-        throw new UnsupportedOperationException(
-            "details() not yet implemented"
-        );
+    public String details() throws IOException {
+        final String dtls = new IoCheckedScalar<String>(
+            () -> new ItemAt<>(
```

Comment:
@carlosmiranda you should be able to pass the object `ItemAt` that implements `Scalar` directly here instead of the lambda

Reply:
@victornoel explained here: https://github.com/zold-io/java-api/pull/45#discussion_r205359262